public/index.html

```diff
@@ -9,7 +9,6 @@
       name="description"
       content="Democratizing credit investing with a positive impact"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
why does this need to go?

Reply:
I removed the logo as it was a generic react one and if I leave this line then it complains in the console.